packages/components/src/Emphasis/Emphasis.component.js

```diff
@@ -0,0 +1,31 @@
+import React, { PropTypes } from 'react';
+import theme from './Emphasis.scss';
+
+function emphasise(text, value) {
+	if (!text) {
+		return '';
+	}
+	if (!value) {
+		return text;
+	}
+
+	const parts = text.split(new RegExp(`(${value})`, 'gi')).filter(Boolean);
+	return parts.map((part, index) => {
+		if (value && part.toUpperCase() === value.toUpperCase()) {
+			return <em key={index} className={theme.highlight}>{part}</em>;
+		}
+		return part;
+	});
+}
+
+function Emphasis(props) {
+	const { value, text } = props;
```

Comment:
Please don't spread those values. Use them directly `this.props.value` and `this.props.text`